Gems/Atom/Tools/MaterialCanvas/Assets/MaterialCanvas/TestData/clamp.materialcanvasnode.azasset

```diff
@@ -0,0 +1,105 @@
+{
+    "Type": "JsonSerialization",
+    "Version": 1,
+    "ClassName": "DynamicNodeConfig",
+    "ClassData": {
+        "category": "Math Functions",
+        "title": "clamp",
+        "inputSlots": [
+            {
+                "name": "inValue1",
```

Comment:
Suggest parameter names like `value`, `valueMin` and `valueMax`.

Reply:
Once we promote this out of being “test data” or we release a preview we can do a usability/UX pass and update all of the node and slot titles, subtitles, variable names, color coding, tool tips, etc. Since I generated all of these and the changes are going to be volatile for the near term improving the code generation I would rather leave everything generic because the data will probably get stomped.